OTAnalytics/application/use_cases/flow_statistics.py

```diff
@@ -0,0 +1,25 @@
+from OTAnalytics.application.use_cases.get_road_user_assignments import (
+    GetRoadUserAssignments,
+)
+from OTAnalytics.domain.flow import FlowId, FlowRepository
+
+
+class NumberOfTracksAssignedToEachFlow:
+    def __init__(
+        self, get_assignments: GetRoadUserAssignments, flow_repository: FlowRepository
+    ) -> None:
+        self._get_assignments = get_assignments
+        self._flow_repository = flow_repository
+
+    def get(self) -> dict[FlowId, int]:
+        result = self._tracks_assigned_to_flows()
```

Comment:
Why do we not just use `defaultdict`?
If we need the method, please find a better name.

Reply:
We could use a defaultdict. But we still need to iterate over all existing flows first, since there can flows that do not have any tracks assigned to. Populating the dictionary with all existing flows ensures that no flows are missing in the result.

Reply:
We only need to add 0 for all flows, if we iterate over the returned dictionary or if we want to export it as a whole.  For the current usage by the ui, we request the number per flow-id by iterating over all flow-ids in the flow-repository, see `frame_flows.py` in line 136. If we request the count for a missing flow-id this will fail. 
As both method iterate over all flows in the flow repository, there is actually no problem. But it might change it the future. I therefore vote to use a `defaultdict` to lower the coupling between the ui class and the use case.